packages/orbit-components/src/Itinerary/ItinerarySegment/index.tsx

```diff
@@ -57,7 +57,9 @@ const ItinerarySegment = ({
             value={{
               index: i,
               opened,
-              toggleOpened: () => setOpened(prev => !prev),
+              toggleOpened: () => {
+                if (document.getSelection()?.type !== "Range") setOpened(prev => !prev);
```

Comment:
maybe add a check here for the document, so on SSR it won't have a type error 

```suggestion
                if (document && document.getSelection()?.type !== "Range") setOpened(prev => !prev);
```